CommerceCloud/Integration/03_SalesforceParticularities.md

```diff
@@ -0,0 +1,170 @@
+
+![Price books](../../Assets/Screenshots/Channels/Settings/Connections/CommerceCloud/PriceBooks.png "[Site-specific attribute definitions]")
+
+
+## Variations
+
+Variants in Actindo must not be double, but this is possible in Salesforce. Therefore, it may be necessary to create a further variant level in Actindo so that there are no double variants. However, this variant level should not be displayed in Salesforce. For this purpose, you have to specify the IDs of all attributes that should not be uploaded as defining attributes in Salesforce when configuring the connection settings, see [Configure the Salesforce connection](./02_ManageSalesforceConnection.md#configure-the-salesforce-connection). This way the value will be uploaded to Salesforce, but not created as a defining attribute. 
+Any information about the Salesforce variants is displayed in the variation products list in Salesforce:    
+*Merchant Tools > Products and Catalogs > Products > Select product > Tab Variations*
+
+### Translatable variants
+
+Basically, variants in Salesforce are always translatable, but they must not be localizable. However, it is possible to specify multiple languages for variants in Salesforce. In Actindo, variants cannot be multi-language. Consequently, Salesforce can specify multilingual values for defining attributes, but Actindo cannot.   
+To be able to make the defining attributes translatable in Actindo anyway, you must do the following:
+- The corresponding variant attributes must be specified in the *PIM* module as tree node attributes.
+- The translations must be specified in the tree node attribute.
+- The *Tree-To-String defining values (CommerceCloud)* extension must be selected to map the destination attribute from Salesforce to the PIM source attribute in the corresponding ETL mapping(s), see [Configure the translatable variants](02_ManageSalesforceConnection.md#configure-the-translatable-variants).
+
+    > [Info] As the destination attribute is a variant defining attribute, it is always a single-language attribute with the *String* data type, whereas the source attribute must be a tree node attribute.   
+
+Thereby, the translations can be imported from the tree-node attribute when uploading and the variants can be made translatable.
+
+![Translatable variants](../../Assets/Screenshots/DataHub/Settings/ETL/CommerceCloud/TranslatableVariants.png "[Translatable variants]")
+
+
+## Product categories
+
+You can create new product categories in Salesforce as well as in Actindo. When uploading a product category from Actindo to Salesforce, the existing product categories are not overwritten, but merely added to the existing categories in Salesforce. Further, only categories that are created and added by Actindo can also be deleted by Actindo. All categories, that are created or edited by Salesforce, are not touched by Actindo at all. Only the changes made in Actindo that do not affect the Salesforce categories are uploaded.
```

Comment:
I would write "category assignment" and not "categories"

Otherwise people might understand that we delete the actual categories